content/src/ports/contentStorage/S3ContentStorage.ts

```diff
@@ -0,0 +1,104 @@
+import AWS from 'aws-sdk'
+import { Readable } from 'stream'
+import { ContentItem, ContentStorage, SimpleContentItem } from './contentStorage'
+
+export async function createS3ContentStorage(
```

Comment:
shouldn't we remove this s3 implementation? it is no longer used and can be restored if needed in the future